content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -39,6 +39,11 @@ Resource quotas work like this:
   See the [walkthrough](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)
   for an example of how to avoid this problem.
 
+{{< note >}}
+- For cpu/memory: ResourceQuota works only if every other pods in the namespace sets a limit for that resource. It means that you cannot create a new pod without limit/request CPU if the resource quota limits the CPU quota of this namespace.
```

Comment:
Does this include hugepage memory resources?

Reply:
@sftim I am not so sure on hugepage memory resources please let me check.

Reply:
Only CPU and memory, see
https://github.com/kubernetes/kubernetes/issues/112310#issuecomment-1245072299.